patches/broccoli-merge-files@0.8.0.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/index.js b/index.js
+index 8844cc54ceb34b77e2ca652eb85452d49db6ab7f..ecfd7eeb45dd96eeb7e05456ef3affa5886fa197 100644
+--- a/index.js
++++ b/index.js
+@@ -44,7 +44,7 @@ class BroccoliMergeFiles extends broccoli_plugin_1.default {
+     async readFiles() {
+         const streams = this.inputPaths.map((path) => [
+             path,
+-            fast_glob_1.default.stream(this.options.patterns, Object.assign({}, this.options.globOptions, { cwd: path, absolute: false, unique: true, onlyFiles: true, onlyDirectories: false, transform: null, stats: false }))
++            fast_glob_1.default.stream(this.options.patterns, Object.assign({}, this.options.globOptions, { cwd: path, absolute: false, unique: true, onlyFiles: true, onlyDirectories: false, stats: false }))
```

Comment:
In `fast-glob@v3`, `transform` is no longer a supported option so I removed it based on the current value, `null`.